### ISSUE TITLE
Discuss other languages

### ISSUE THREAD

**Issue by rabernat** [pangeo-data/pangeo-cmip6-cloud#14]
Right now these docs only refer to python. However, the data is accessible from any language that can read Zarr / CSV. I would really prefer to broaden these docs to discuss those options. We can go into more detail on the python tools, but when I read
>  Interaction with the data is done primarily through Python, using intake, intake-esm, and xarray.

I see users from other languages immediately being disappointed.

**Comment by charlesbluca** [comment]
I agree, I'll make alterations to the main page to focus on the back end formats being used (Zarr, CSV) and take some time in the overview page describing Zarr to list out the various languages that support it and their associated projects. Is there a master list of the supported languages somewhere? Couldn't find anything through the main documentation.

Would be useful to add some simple examples of opening a Zarr store in some of the various languages? We already have [Julia covered](https://github.com/pangeo-data/pangeo-julia-examples/blob/master/CMIP6-GCS-Julia.ipynb) but I know there are several other supported languages.

**Comment by charlesbluca** [comment]
Merged #15 and added information on other languages supporting Zarr/CSV; a potential next step would be to include some code examples for other languages, if any examples are available or if anyone has experience with some of the other Zarr packages.
